### PR TITLE
Option for rebuilding all data at once

### PR DESCRIPTION
**Pull request by cytl**
It's very helpful when there is a option for rebuilding **all** data at once.

### CODE REVIEW

File: wcfsetup/install/files/acp/templates/rebuildData.tpl [hunk 1]
```diff
@@ -35,7 +35,9 @@
 		
 		{foreach from=$objectTypes item=objectType}
 			<dl class="wide">
-				<dd>
+			                <a class="button small" id="rebuildAllData">{lang}wcf.acp.rebuildAllData{/lang}</a>
```
Comment (cytl) [inline wcfsetup/install/files/acp/templates/rebuildData.tpl]: Added button

**Comment by TimWolla** [comment]
Is this pull request complete? I wonder how this should work.

File: wcfsetup/install/files/acp/templates/rebuildData.tpl [hunk 2]
```diff
@@ -35,7 +35,9 @@
 		
 		{foreach from=$objectTypes item=objectType}
 			<dl class="wide">
-				<dd>
```
Comment (max-m) [inline wcfsetup/install/files/acp/templates/rebuildData.tpl]: This line should not be removed.

**Comment by cytl** [comment]
Sure, not at all. I will deliver the full code within the next few days.

**Comment by max-m** [comment]
So it is incomplete …

**Comment by dtdesign** [comment]
Feel free to submit a new pull request once everything is in place. Additionally please try to submit as few commits as possible.

**Pull request closed without merging**